src/components/Reports/AdditionalSalaryRequest/Shared/AdditionalSalaryRequestContext.tsx

```diff
@@ -15,6 +19,8 @@ export type AdditionalSalaryRequestType = {
   setIsDrawerOpen: (open: boolean) => void;
   handleCancel: () => void;
   handleBack: () => void;
+  percentComplete: number;
+  formik: FormikProps<CompleteFormValues>;
```

Comment:
Instead of adding Formik props to this context, you can get them with `useFormikContext()`.